pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/flink.apache.org/v1beta1/FlinkDeployment/customizations_tests.yaml

```diff
@@ -2,9 +2,116 @@ tests:
   - desiredInputPath: testdata/desired-flinkdeployment.yaml
     statusInputPath: testdata/status-file.yaml
     operation: AggregateStatus
-  - observedInputPath: testdata/observed-flinkdeployment.yaml
+    desiredResults:
```

Comment:
Can the output content also be carried in a file format, just like the input content, to maintain the current file's simplicity?

Reply:
I think this is expected; otherwise, this file would become very large and it would be difficult to see how we organized the test cases.